Dizzy/Screens/PlaceStory/PlaceStoryVM.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  PlaceStoryVM.swift
+//  Dizzy
+//
+//  Created by Tal Ben Asuli on 25/04/2019.
+//  Copyright © 2019 Dizzy. All rights reserved.
+//
+
+import Foundation
+
+protocol PlaceStoryVMDelegate: class {
+    func placeStoryVMDidFinised(_ viewModel: PlaceStoryVMType)
+}
+
+protocol PlaceStoryVMType {
+    var showImage: (String) -> Void { get set }
```

Comment:
why wont you make this observable called imageUrl? the name showImage fells like boolean and its not